pkgdown/_pkgdown.yml

```diff
@@ -83,6 +100,11 @@ reference:
   contents:
   - dm_nycflights13
   - dm_nrow
+  - dm_disambiguate_cols
+  - dm_paste
+  - dm_select
+  - dm_rename
```

Comment:
Move this to a separate topic, close to "Filtering"?

Reply:
Ok, I can move them. Just out of curiosity, why close to "Filtering"?

Reply:
Filtering and selection are two basic operations that are invoked first when using a ready-made dm as a consumer.

Reply:
Name of the new category: "Rename or select columns of tables in a `dm`"

Open to constructive criticism :)